geoportailv3/templates/index.html

```diff
@@ -325,6 +330,7 @@ <h2 translate>infos</h2>
          appModule.value('appShorturlTemplateUrl', 'share/shorturl.html');
          appModule.value('appQueryTemplateUrl', 'query/info.html');
          appModule.value('appLocationinfoTemplateUrl', 'locationinfo/locationinfo.html');
+         appModule.value('appMymapsTemplateUrl', 'mymaps/mymaps.html');
```

Comment:
This will conflict with https://github.com/Geoportail-Luxembourg/geoportailv3/pull/848.